client/client_main.cpp

```diff
@@ -374,7 +374,9 @@ int client_main(int argc, char *argv[])
        {{"S", "Sound"}, _("Read sound tags from FILE."), "FILE"},
        {{"m", "music"}, _("Read music tags from FILE."), "FILE"},
        {{"t", "tiles"}, _("Use data file FILE.tilespec for tiles."), "FILE"},
-       {{"w", "warnings"}, _("Warn about deprecated modpack constructs.")}});
+       {{"w", "warnings"}, _("Warn about deprecated modpack constructs.")},
+       {"debug-tileset", _("Prints tileset debugging information.")}});
```

Comment:
I don't see this in compiled code

```
$ build_2346/freeciv21-client --help
Encodings: Data=UTF-8, Local=en_US, Internal=UTF-8
Usage: build_2346/freeciv21-client [options] [url]

Options:
  -h, --help              Displays help on commandline options.
  --help-all              Displays help including Qt specific options.
  -v, --version           Displays version information.
  -A, --Announce <PROTO>  Announce game in LAN using protocol PROTO
                          (IPv4/IPv6/none).
  -a, --autoconnect       Skip connect dialog.
  -d, --debug <LEVEL>     Set debug log level
                          (fatal/critical/warning/info/debug).
  -F, --Fatal             Raise a signal on failed assertion.
  -f, --file <FILE>       Load saved game FILE.
  -H, --Hackless          Do not request hack access to local, but not spawned,
                          server.
  -l, --log <FILE>        Use FILE as logfile (spawned server also uses this).
  -M, --Meta <HOST>       Connect to the metaserver at HOST.
  -n, --name <NAME>       Use NAME as username on server.
  -p, --port <PORT>       Connect to server port PORT (usually with -a).
  -P, --Plugin <PLUGIN>   Use PLUGIN for sound output.
  -r, --read <FILE>       Read startup script FILE (for spawned server only).
  -s, --server <HOST>     Connect to the server at HOST (usually with -a).
  -S, --Sound <FILE>      Read sound tags from FILE.
  -m, --music <FILE>      Read music tags from FILE.
  -t, --tiles <FILE>      Use data file FILE.tilespec for tiles.
  -w, --warnings          Warn about deprecated modpack constructs.

Arguments:
  url                     Server information in URL format.
```

Reply:
I do...
```
$ build/freeciv21-client --help
Encodings: Data=UTF-8, Local=fr_BE, Internal=UTF-8
Utilisation : build/freeciv21-client [options] [url]

Options :
  -h, --help              Displays help on commandline options.
  --help-all              Displays help including Qt specific options.
  -v, --version           Afficher les informations de version.
  -A, --Announce <PROTO>  Announce game in LAN using protocol PROTO
                          (IPv4/IPv6/none).
  -a, --autoconnect       Skip connect dialog.
  -d, --debug <LEVEL>     Set debug log level
                          (fatal/critical/warning/info/debug).
  -F, --Fatal             Raise a signal on failed assertion.
  -f, --file <FILE>       Load saved game FILE.
  -H, --Hackless          Do not request hack access to local, but not spawned,
                          server.
  -l, --log <FILE>        Use FILE as logfile (spawned server also uses this).
  -M, --Meta <HOST>       Connect to the metaserver at HOST.
  -n, --name <NAME>       Use NAME as username on server.
  -p, --port <PORT>       Connect to server port PORT (usually with -a).
  -P, --Plugin <PLUGIN>   Use PLUGIN for sound output.
  -r, --read <FILE>       Read startup script FILE (for spawned server only).
  -s, --server <HOST>     Connect to the server at HOST (usually with -a).
  -S, --Sound <FILE>      Read sound tags from FILE.
  -m, --music <FILE>      Read music tags from FILE.
  -t, --tiles <FILE>      Use data file FILE.tilespec for tiles.
  -w, --warnings          Warn about deprecated modpack constructs.
  --debug-tileset         Prints tileset debugging information.

Arguments :
  url                     Server information in URL format.
  ```

Reply:
I started a fresh test from scratch. hm...

Reply:
I should note that the start up noise is reduced, so some of the changes are working.